pages/home.py

```diff
@@ -240,32 +270,100 @@ def update_model(contents, filename, df, column, about_us):
                 ])
             )
-            plot_component.append(
+            weights_plots.append(
                 dbc.Row([
                     dbc.Col([tbl_metrics_adj_ensemble(model, X, y, task, library, weights)], width=4)
                 ], justify="center")
             )
 
         for plot in metrics.permutation_feature_importance_all(model, X, y, library=library, task=task):
-            plot_component.append(dcc.Graph(figure=plot, className="plot"))
+            metrics_plots.append(dcc.Graph(figure=plot, className="plot"))
+
+        metrics_plots.append(
+            html.H2("""
+                Partial Dependence isolate one specific feature's effect on the model's output while maintaining 
+                all other features at fixed values. It capturing how the model's output changes as the chosen 
+                feature varies. 
+                """,
+                className="annotation_str", id="ann_1")
+        )
 
         for plot in metrics.partial_dependence_plots(model, X, library=library, autogluon_task=task):
-            plot_component.append(dcc.Graph(figure=plot, className="plot"))
+            metrics_plots.append(dcc.Graph(figure=plot, className="plot"))
 
         # It may be necessary to keep the model for the code with weights,
         # for now we remove the model after making charts
         try:
             shutil.rmtree('./uploaded_model')
         except FileNotFoundError:
             pass
-        children = html.Div(plot_component)
 
-    return children, model_names, predictions, task
+        metrics_plots.insert(0, html.Div([
+            dbc.Row([
+                dbc.Col([html.Button('Weights', id="weights", className="button_1")], width=2),
+                dbc.Col([html.Button('Metrics', id="metrics", className="button_1")], width=2),
+                dbc.Col([html.Button('Compatimetrics', id="compatimetrics", className="button_1")], width=2),
+            ], justify="center"),
+        ], style={"display": "block", "position": "sticky"}))
+        weights_plots.insert(0, html.Div([
+            dbc.Row([
+                dbc.Col([html.Button('Weights', id="weights", className="button_1")], width=2),
+                dbc.Col([html.Button('Metrics', id="metrics", className="button_1")], width=2),
+                dbc.Col([html.Button('Compatimetrics', id="compatimetrics", className="button_1")], width=2),
+            ], justify="center"),
+        ], style={"display": "block", "position": "sticky"}))
+
+        weights_plots = html.Div(weights_plots)
+        children = html.Div(metrics_plots)
+
+    return children, children, weights_plots, model_names, predictions, task
+
+
+# callbacks for buttons to change plots categories
+@callback(
+    Output('plots', 'children', allow_duplicate=True),
+    Input('weights', 'n_clicks'),
+    State('weight_plots', 'data'),
+    State('plots', 'children'),
+    prevent_initial_call=True
+)
+def show_weights(n_clicks, data, children):
```

Comment:
This functions throw annoying warning which is not crucial to app functionality, but I change this in next commit.